StarVMC/StarAgmlLib/AgMLStructure.h

```diff
@@ -220,7 +220,12 @@ class AgMLStructure : public AgMLStructureBase
 public:
 
   AgMLStructure( const char *_name ) : AgMLStructureBase(), name(_name) { };
-
+ ~AgMLStructure() {   
+   for ( auto t : table ) { // cleanup the table
+     if (t) delete (t);
```

Comment:
```suggestion
     delete t;
```

delete works for nullptr's

Reply:
I may be over / underthinking the problem here... but are we guaranteed that we won't recieve a zero from TTable?